package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint delegated-events.js test",
     "bootstrap": "git submodule update --init && npm install",
     "prebuild": "npm run lint",
-    "build": "rollup -c rollup.config.dist.js",
+    "build": "rollup -c rollup.config.dist.js && cp delegated-events.js.flow dist/",
```

Comment:
Right, delegated-events ships with two "main"s: An AMD source file at `dist/delegated-events.js` and an ES6 file at `delegated-events.js`. Therefore we need to provide the types for the generated AMD wrapper as well.